README.md

```diff
@@ -164,6 +164,35 @@ CHROME=y rspec
 COVERAGE=y rspec
 ```
 
+#### Percy visual diff tests
```

Comment:
Maybe this should move into a `doc/` file so that `README.md` is a little shorter. Not a blocker.

It can be without a datestamp since it's currently evergreen. Maybe `doc/visual-diff-testing.md`?